metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py

```diff
@@ -854,7 +862,12 @@ def gen_dataset_workunits(
             else None,
             lastModified=TimeStamp(time=int(table.last_altered.timestamp() * 1000))
             if table.last_altered is not None
+            else TimeStamp(time=int(table.created.timestamp() * 1000))
+            if table.created is not None
             else None,
+            externalUrl=BQ_EXTERNAL_TABLE_URL_TEMPLATE.format(
```

Comment:
minor: let's add the ability to disable this via a config. Snowflake connector _already_ has this. please reuse the same convention!